### PR TITLE
release notes for v0.11.0

### PR DESCRIPTION
**Pull request by 3timeslazy**

### CODE REVIEW

**Review by ziflex — CHANGES_REQUESTED** [review]
1. Add notes about COVID-19
2. Remove notion of Ferret's team participation in OpenVirus and announcement of Worker. These topics deserve their own blog posts.

File: content/blog/ferret-v0.11.0.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+LET favicon = DOWNLOAD('https://www.google.com/favicon.ico')
+
+// 'WRITE' the same as 'IO::FS::WRITE'.
+// RETURN IO::FS::WRITE('google.favicon.ico', favicon) also valid.
+RETURN WRITE('google.favicon.ico', favicon)
+{{</ code >}}
+
+## PATH functions
+A little bit functions to work with path strings.
```
Comment (ziflex) [inline content/blog/ferret-v0.11.0.md]: ``New functions for working with file paths.``

File: content/blog/ferret-v0.11.0.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+{{</ code >}}
+
+---
+
+# What's fixed
+## RANDOM_TOKEN
+``RANDOM_TOKEN`` return random string now! (even on Windows)
+
+## IO::FS::WRITE
+Now there is no need to give the created files read access manually.
```
Comment (ziflex) [inline content/blog/ferret-v0.11.0.md]: ``Now files are created with read access automatically.``

File: content/blog/ferret-v0.11.0.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+---
+# Around the Ferret
+## OSS-Fuzz platform
+We are tried to connect Ferret with [OSS-Fuzz](https://github.com/google/oss-fuzz) platform. But Google has [rejected](https://github.com/google/oss-fuzz/pull/3782) Ferret :(
```
Comment (ziflex) [inline content/blog/ferret-v0.11.0.md]: ``We have tried...``

File: content/blog/ferret-v0.11.0.md (hunk 1 above)
Comment (3timeslazy) [reply content/blog/ferret-v0.11.0.md]: done

File: content/blog/ferret-v0.11.0.md (hunk 2 above)
Comment (3timeslazy) [reply content/blog/ferret-v0.11.0.md]: done

File: content/blog/ferret-v0.11.0.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+author: "Vladimir Fetisov"
+authorLink: "https://github.com/3timeslazy"
+date: "2020-05-19"
+---
+
+Hello fellow miners, **[Ferret v0.11.0](https://github.com/MontFerret/ferret/releases/tag/v0.11.0)** has been **finally** released!
+
+In this release, we mostly focused on our full-time jobs and families.
+
+But despite this, We have done a lot of cool features and projects.
```
Comment (ziflex) [inline content/blog/ferret-v0.11.0.md]: ``we...``

File: content/blog/ferret-v0.11.0.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+Let’s take a look!
+
+--- 
+But there is one word before starting:
```
Comment (ziflex) [inline content/blog/ferret-v0.11.0.md]: Remove the ``But there is one word before starting:``.
And move the entire intro to the top. (Let's start the RN with this notion of COVID19)

**Review by ziflex — CHANGES_REQUESTED** [review]
Final bits :)

**Review by ziflex — APPROVED** [review]

**Pull request merged**